### DISCUSSION TITLE
Welcome to Rick-and-Morty Discussions!

### DISCUSSION THREAD

**Discussion by amanyih** [amanyih/Rick-and-Morty#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amanyih** [comment]
What other features do you want to be added?

**Reply by Elizura** [reply]
It would be nice if you can add GIFs
